src/foundry/foundry.js/config.d.ts

```diff
@@ -599,7 +599,7 @@ declare global {
       blurStrength: number;
 
       /** @defaultValue `ScreenCulling` */
-      cullingBackend: unknown; // FIXME: ConstructorOf<typeof ScreenCulling>
+      cullingBackend: ConstructorOf<typeof ScreenCulling | typeof QuadtreeCulling>;
```

Comment:
I suppose we could add interfaces
```ts
interface CullingBackend {
  cull(rect: Rectangle): void;
}

interface CullingBackendConstructor {
  new (): CullingBackend;
}
```
and then use `CullingBackendConstructor` as the type for `cullingBackend`?

Reply:
We have no idea though whether it only depends on the `cull` method. `ScreenCulling` also has `intersects`.

Reply:
From a quick search, only the `cull` method is called externally. But I suppose it doesn’t matter a lot, so if you don’t want to do it for now, that’s fine, too.